explorer/templates.go

```diff
@@ -120,6 +166,9 @@ func makeTemplateFuncMap(params *chaincfg.Params) template.FuncMap {
 		"divide": func(n, d int64) int64 {
 			return n / d
 		},
+		"divideFloat": func(n float64, d float64) float64 {
```

Comment:
Please elide the first `float64` as we have with the other `int64` functions.

Reply:
ok. Noted